tests/storage-vm

```diff
@@ -517,6 +517,31 @@ for poolDriver in $poolDriverList; do
         # Finally, unset the value to avoid any potential side effects on other tests.
         lxc config unset instances.migration.stateful
 
+        if hasNeededAPIExtension instances_migration_stateful; then
```

Comment:
Looks like this needs to be quite a few lines before that. See the CI failures.

Reply:
fixed